llama-index-core/llama_index/core/readers/file/base.py

```diff
@@ -63,6 +63,8 @@ def _try_loading_included_file_formats() -> Dict[str, Type[BaseReader]]:
             PptxReader,
             VideoAudioReader,
         )  # pants: no-infer-dep
+
+        from llama_index.readers import JSONReader
```

Comment:
I think you may be using a legacy version of `llama_index` (i.e., v0.9.4x and earlier)?

`JSONReader` is available in our integration package call `llama-index-readers-json` which isn't included as a dependency in the umbrella `llama-index` package. Since we don't include that in that umbrella package, we don't try to import it here.

Reply:
Hey @nerdai , i am using the latest 0.10 version off of main.  Maybe i am confused by I see the JSONreader class in the llama-index-core here ?  https://github.com/run-llama/llama_index/blob/main/llama-index-core/llama_index/core/readers/json.py .  Is that not what I should be using?